src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesOwaspDependencyScan.java

```diff
@@ -111,13 +135,97 @@ private static void checkMaven(LocalRepository repository, ValueSet values) thro
   }
 
   /**
-   * Checks if a project uses OWASP Dependency Check Gradle plugin.
+   * Checks if a project uses OWASP Dependency Check Gradle plugin,
+   * and updates a value set with corresponding feature values.
    *
    * @param repository The project's repository.
-   * @param values set of type {@link ValueSet}.
+   * @param values A {@link ValueSet} to be updated.
+   */
+  private static void checkGradle(LocalRepository repository, ValueSet values) throws IOException {
+    for (Path gradleFile : repository.files(path -> path.getFileName().endsWith(".gradle"))) {
+      Optional<List<String>> something = repository.readLinesOf(gradleFile);
+      if (!something.isPresent()) {
+        return;
+      }
+      List<String> file = something.get();
+
+      if (foundOwaspDependencyCheckInGradle(file)) {
+        boolean isMainFile = "build.gradle".equals(gradleFile.toString().toString());
```

Comment:
Is toString().toString() necessary? Os is this a typo?

Reply:
It's a typo. It's not fixed.